src/adapters/shoebill/subgraph/subgraph.yaml

```diff
@@ -0,0 +1,57 @@
+specVersion: 1.0.0
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: LayerBank
+    network: zklink-nova
+    source:
+      abi: LayerBankCore
+      address: '0x89e348e0ab4D83aEDbf1354004552822a805a12e'
+      startBlock: 2335536
```

Comment:
Set the startBlock to 2335136 as it was created at this [block](https://explorer.zklink.io/tx/0x04f6609ef2c36a1b98ffc27c0e4a34932bcb8b21fc73b27a054d0b2f1e397740)